onecodex/analyses.py

```diff
@@ -306,8 +322,9 @@ def _to_classification_df(
         normalize="auto",
         table_format="wide",
         include_taxa_missing_rank=False,
+        include_nans=False,
```

Comment:
Make kwarg names consistent with `_to_functional_df` (`filler` & `fill_missing`). I think the default will have to be different though for backwards compatibility.